src-tauri/src/app.rs

```diff
@@ -86,18 +87,19 @@ impl Service {
     ///
     /// The node and several channel senders are added to the shared app context while channel
     /// receivers are stored on the Service struct for use during the runtime loop.
-    pub async fn build(blobs_base_dir: PathBuf) -> anyhow::Result<Self> {
-        let private_key = PrivateKey::new();
+    pub async fn build(app_data_dir: PathBuf) -> anyhow::Result<Self> {
```

Comment:
Renamed to `app_data_dir` since blobs and keystore are both now using this as a base path.